tests/event_lists/test_south_african.py

```
@@ -0,0 +1,405 @@
+    "South Africa–Soviet Union relations": "العلاقات الجنوب الإفريقية السوفيتية",
+    "South Africa–Spain military relations": "العلاقات الإسبانية الجنوب الإفريقية العسكرية",
+    "South Africa–Spain relations": "العلاقات الإسبانية الجنوب الإفريقية",
+    "South Africa–Sri Lanka relations": "العلاقات الجنوب الإفريقية السريلانكية",
+    "South Africa–Sri Lanka sports relations": "العلاقات الجنوب الإفريقية السريلانكية الرياضية",
+    "South Africa–Sweden relations": "العلاقات الجنوب الإفريقية السويدية",
+    "South Africa–Switzerland relations": "العلاقات الجنوب الإفريقية السويسرية",
+    "South Africa–Syria relations": "العلاقات الجنوب الإفريقية السورية",
+    "South Africa–Taiwan relations": "العلاقات التايوانية الجنوب الإفريقية",
+    "South Africa–Tanzania relations": "العلاقات التنزانية الجنوب الإفريقية",
+    "South Africa–Thailand relations": "العلاقات التايلندية الجنوب الإفريقية",
+    "South Africa–Thailand sports relations": "العلاقات التايلندية الجنوب الإفريقية الرياضية",
```

Ruff (0.14.10)

Dictionary key literal `"South Africa–Thailand sports relations"` repeated

Remove repeated key literal `"South Africa–Thailand sports relations"`

(F601)